### DISCUSSION TITLE
Mac related question

### DISCUSSION THREAD

**Discussion by Qokblowa** [ppy/osu#17329] [Q&A] [answered]
Is there any way to run osu!lazer on older MacOS versions? It always asks for 10.14+ (I think)

**Comment by frenzibyte** [comment, marked as answer]
The minimum version is macOS 10.15, and no there's no way to run at any macOS version older than that.

![CleanShot 2022-03-19 at 02 38 29](https://user-images.githubusercontent.com/22781491/159097191-58388449-669b-457f-83b8-a118d2886f7a.png)
![CleanShot 2022-03-19 at 02 38 17](https://user-images.githubusercontent.com/22781491/159097172-f7585670-4f0f-4a3a-bed7-ca68ae296ef4.png)
